WordPress/Classes/Services/BlogService.m

```diff
@@ -20,14 +20,28 @@
 #import "NSString+XMLExtensions.h"
 #import "TodayExtensionService.h"
 
+NSString *const LastUsedBlogURLDefaultsKey = @"LastUsedBlogURLDefaultsKey";
+NSString *const BlogEntity = @"Blog";
+NSString *const EditPostViewControllerLastUsedBlogURLOldKey = @"EditPostViewControllerLastUsedBlogURL";
+NSString *const BlogName = @"blogName";
+NSString *const GetFeatures = @"wpcom.getFeatures";
+NSString *const VideopressEnabled = @"videopress_enabled";
+NSString *const XmlRpc = @"xmlrpc";
+NSString *const Standard = @"standard";
+NSString *const MinimumVersion = @"3.6";
+NSString *const DateCreatedGmt = @"date_created_gmt";
+NSString *const TimeZoneName = @"timezone";
```

Comment:
This one, together with `GmtOffset` and `TimeZone` could be defined in Blog.m so we can use it in other sources too.  Again great work on using constants!